client/verta/verta/_registry/modelversion.py

```diff
@@ -28,6 +28,28 @@
 
 
 class RegisteredModelVersion(_ModelDBEntity):
+    """
+    Object representing a version of a Registered Model.
+
+    There should not be a need to instantiate this class directly; please use
+    :meth:`RegisteredModel.get_or_create_version`.
+
+    Attributes
+    ----------
+    id : int
+        ID of this Model Version.
+    name : str
+        Name of this Model Version.
+    has_environment : bool
+        Whether there is an environment associated with this Model Version
```

Comment:
Super minor: periods at the ends of all of the descriptions, please!
```suggestion
        Whether there is an environment associated with this Model Version.
```

Reply:
Added.